src/main/groovy/liquibase/harness/change/ChangeObjectTestHelper.groovy

```diff
@@ -36,11 +36,12 @@ class ChangeObjectTestHelper {
         Logger.getLogger(this.class.name).warning("Only " +  TestConfig.instance.inputFormat + " input files are taken into account for this test run")
 
         List<TestInput> inputList = new ArrayList<>()
-        for (DatabaseUnderTest databaseUnderTest : TestConfig.instance.databasesUnderTest) {
+
+        for (DatabaseUnderTest databaseUnderTest : TestConfig.instance.initializeDatabasesConnection(TestConfig.instance.databasesUnderTest)) {
```

Comment:
can we move initializeDatabasesConnection() to other class now? Seems like now it's not tied up to TestConfig as much. And structures like  `TestConfig.instance.initializeDatabasesConnection(TestConfig.instance.databasesUnderTest)` looks weird

Reply:
Fixed.